--- virtuoso.ini	2016-06-01 12:49:35.872924678 +0300
+++ virtuoso.ini.new	2016-06-01 12:52:00.058319710 +0300
@@ -63,7 +63,7 @@
 CheckpointAuditTrail		= 0
 AllowOSCalls			= 0
 SchedulerInterval		= 10
-DirsAllowed			= ., /usr/share/virtuoso/vad
+DirsAllowed			= ., /usr/share/virtuoso/vad, /models
 ThreadCleanupInterval		= 0
 ThreadThreshold			= 10
 ResourcesCleanupInterval	= 0
@@ -113,8 +113,8 @@
 ;; Note the default settings will take very little memory
 ;; but will not result in very good performance
 ;;
-NumberOfBuffers          = 10000
-MaxDirtyBuffers          = 6000
+NumberOfBuffers          = 85000
+MaxDirtyBuffers          = 65000
 
 
 [HTTPServer]
